wgpu-hal/src/vulkan/conv.rs

```diff
@@ -1,5 +1,10 @@
 use ash::vk;
 
+// https://registry.khronos.org/vulkan/specs/1.3-extensions/man/html/VkImageAspectFlagBits.html
+const VK_IMAGE_ASPECT_PLANE_0_BIT: vk::Flags = 0x00000010;
+const VK_IMAGE_ASPECT_PLANE_1_BIT: vk::Flags = 0x00000020;
+const VK_IMAGE_ASPECT_PLANE_2_BIT: vk::Flags = 0x00000040;
```

Comment:
These constants are available in ash: https://docs.rs/ash/0.37.3+1.3.251/ash/vk/struct.ImageAspectFlags.html#associatedconstant.PLANE_0_KHR

Reply:
Thanks.